webpack.config.js

```diff
@@ -22,10 +20,10 @@ const config = {
     },
     entry: './src/extension.ts', // the entry point of this extension, 📖 -> https://webpack.js.org/configuration/entry-context/
     output: { // the bundle is stored in the 'dist' folder (check package.json), 📖 -> https://webpack.js.org/configuration/output/
-        path: path.resolve(__dirname, 'dist', 'scripts'),   // the clipboardy package ships with binaries in the fallbacks folder and reference them with "../fallbacks/*"
+        path: path.resolve(__dirname, 'dist'),
```

Comment:
Any impact on user upgrade the version?

Reply:
I don't think so. Extension packages are replaced as a whole.